install.sh

```diff
@@ -167,5 +167,55 @@ runchecksonstart: true
 loglevel: error
 EOF
 
-printSuccess "The Site Wide Analysis Tool Agent has been successfully installed at $agentPath"
-[ "$installDaemon" ] && installAndConfigureDaemon || installAndConfigureCron
+echo "** Install validation "
+
+if [[ ! -f "$appRoot/app/etc/env.php" ]]; then
+  error_exit "Magento not found."
+else
+  echo "Magento Found - OK"
+fi
+
+if nc -z $updaterDomain 443 2>/dev/null; then
+    echo "Connect to API Server - OK"
+else
+    error_exit "Can not connect to API Server"
+fi
+
+if [ -f "$agentPath/config.yaml" ]; then
+    echo "Config File is created - OK"
+else
+    error_exit "Config File was not created."
+fi
+
+phpVersion=$($phpPath -v | awk '{ print $2 }' | head -1)
+semver=( ${phpVersion//./ } )
+major="${semver[0]}"
+minor="${semver[1]}"
+
+echo "**Checking php version."
+
+if [ "$major" -eq 7 ]; then
```

Comment:
As far I heard we going to support PHP8

Reply:
It will not fail, this condition is specifically for php7